docs/position_inversion.md

```diff
@@ -9,20 +9,27 @@ the Seafloor Geodesy Community experiment.
 
 The first step is to prepare all of the input files required to calculate a
 position, including the _pxp_tt_ file with acoustic two-way travel times, the
-_POS_FREED_TRANS_TWTT_ file with transducer positions, and the sound velocity
-profile. Create a working directory and place a configuration file named
-_config.yaml_ inside of it. Edit _config.yaml_ to reflect the a priori site
-information and locations of the input data files. Once you have done this, run
-GNATSS in the working directory with the following command on the command line
-interface:
+_gps_positions_ file with antenna positions, the Novatel files with #INSPVA and
+#INSSTDEVA strings, and the sound velocity profile. Create a working directory
+and place a configuration file inside of it. This configuration file may be
+edited from the template available on the [GNATSS Config File](./config_yaml.md)
+page or the [GNATSS GitHub](https://github.com/seafloor-geodesy/gnatss). You may
+name this file whatever you like, but for the purposes of this example, we will
+name the configuration file _NDP1_2022.yaml_. Edit _NDP1_2022.yaml_ to reflect
+the a priori site information and locations of the input data files. Once you
+have done this, run GNATSS in the working directory with the following command
+on the command line interface:
 
 ```
-gnatss run --extract-dist-center --extract-process-dataset --qc --distance-limit 150
+gnatss run --distance-limit 150 NDP1_2022.yaml
 ```
 
-After a successful run, GNATSS will generate an _output_ folder with the
-following files:
+After a successful run, GNATSS will generate an _output_ directory defined in
+the configuration file, by default _./output_. The output directory will contain
+the following files:
 
+- _gps_solution.csv_: Input data file in the Community Standard GNSS-A format.
```

Comment:
```suggestion
- `gps_solution.csv`: Input data file in the Community Standard GNSS-A format.
```

Change these to be small code stylings to stand out more.

Reply:
Done.